src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -571,35 +524,150 @@ def get_worker_id(self) -> str:
-                ssm_param_name = (
-                    f"/aws/service/ami-amazon-linux-latest/{EC2InstanceWorker.AL2023_AMI_NAME}"
-                )
-            else:
-                # Grab the latest Windows Server 2022 AMI
-                # https://aws.amazon.com/blogs/mt/query-for-the-latest-windows-ami-using-systems-manager-parameter-store/
-                ssm_param_name = (
-                    f"/aws/service/ami-windows-latest/{EC2InstanceWorker.WIN2022_AMI_NAME}"
-                )
 
-            response = call_api(
-                description=f"Getting latest {self.configuration.operating_system.name} AMI ID from SSM parameter {ssm_param_name}",
-                fn=lambda: self.ssm_client.get_parameters(Names=[ssm_param_name]),
+@dataclass
+class PosixInstanceWorker(EC2InstanceWorker):
+
+    def ssm_document_name(self) -> str:
+        return "AWS-RunShellScript"
+
+    def _start_worker_agent(self) -> None:
+        assert self.instance_id
+        LOG.info(f"Sending SSM command to configure Worker agent on instance {self.instance_id}")
+
+        cmd_result = self.send_command(
+            f"{self.configure_worker_command(config=self.configuration)}"
+        )
+        assert cmd_result.exit_code == 0, f"Failed to configure Worker agent: {cmd_result}"
+        LOG.info("Successfully configured Worker agent")
+
+        LOG.info(f"Sending SSM command to start Worker agent on instance {self.instance_id}")
+        cmd_result = self.send_command(
+            " && ".join(
+                [
+                    f"nohup runuser --login {self.configuration.agent_user} -c 'AWS_DEFAULT_REGION={self.configuration.region} deadline-worker-agent > /tmp/worker-agent-stdout.txt 2>&1 &'",
+                    # Verify Worker is still running
+                    "echo Waiting 5s for agent to get started",
+                    "sleep 5",
+                    "echo 'Running pgrep to see if deadline-worker-agent is running'",
+                    f"pgrep --count --full -u {self.configuration.agent_user} deadline-worker-agent",
+                ]
+            ),
+        )
+        assert cmd_result.exit_code == 0, f"Failed to start Worker agent: {cmd_result}"
+        LOG.info("Successfully started Worker agent")
+
+        self.worker_id = self.get_worker_id()
+
+    def configure_worker_command(
+        self, config: DeadlineWorkerConfiguration
+    ) -> str:  # pragma: no cover
+        """Get the command to configure the Worker. This must be run as root."""
+
+        cmds = [
+            config.worker_agent_install.install_command_for_linux,
+            *(config.pre_install_commands or []),
+            # fmt: off
+            (
+                "install-deadline-worker "
+                + "-y "
+                + f"--farm-id {config.farm_id} "
+                + f"--fleet-id {config.fleet.id} "
+                + f"--region {config.region} "
+                + f"--user {config.job_user} "
```

Comment:
This should be the agent user

```suggestion
                + f"--user {config.agent_user} "
```